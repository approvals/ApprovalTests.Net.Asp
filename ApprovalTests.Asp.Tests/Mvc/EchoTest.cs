using ApprovalTests.Asp.Mvc;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ApprovalTests.Asp.Tests.Mvc
{
    [TestClass]
    public class EchoTest 
    {
        [TestMethod]
        public void TestBootStrapReady()
        {
            MvcApprovals.VerifyApprovalBootstrap();
        }
    }
}
