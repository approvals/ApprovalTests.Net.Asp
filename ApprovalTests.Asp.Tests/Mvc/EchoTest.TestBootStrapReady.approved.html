<pre> 
If you are seeing this then ApprovalTests.Asp.Bootstrap is setup correctly

Echoing 
'Testing123'

</pre>